package.json

```diff
@@ -22,8 +22,12 @@
     "ts-node": "^10.9.2"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^6.4.1",
+    "@jest/globals": "^29.7.0",
```

Comment:
actually you don't need this package because you're not using it.
if you remove you might have to update the jest.setup.ts to the path without /jest-globals suffix.
if you were using this package you would have to import {describe, it, expect} in your tests which you don't have to. this is because @testing-library/jest-dom comes with @types/jest and you're also declaring @types/jest. it's either @types/jest or @jest/globals